CHANGELOG.md

```diff
@@ -1,12 +1,17 @@
 
 # CHANGELOG
 
+## v11.1.0.0
+
+* Upgrade to [Cumulus v11.1.0](https://github.com/nasa/Cumulus/releases/tag/v11.1.0)
+  * see [Cumulus v11.0.0](https://github.com/nasa/Cumulus/releases/tag/v11.0.0) release notes for required migration steps for workflows and collection configurations
```

Comment:
See my comment on the changelog in the CIRRUS-core PR - https://github.com/asfadmin/CIRRUS-core/pull/129#discussion_r847628207